### PR TITLE
Added Browser Mentions Recipe

### PR DESCRIPTION
**Pull request by sephorus**
Recipe to find how many times web browsers were mentioned on a webpage

### CODE REVIEW

File: src/recipes/browserMentions.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
```
Comment (gregwhitworth) [inline src/recipes/browserMentions.js]: There needs to be a test included with this.

Comment (JRPortfolio) [inline src/recipes/browserMentions.js]: Added test page in 3cb91c692856e0af954778feb4a0a8d28af0cf2e

File: src/recipes/browserMentions.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* 
+    RECIPE: browserMentions
+    -------------------------------------------------------------
+    Author: Morgan, Malick, Lia, Joel
+    Description: Finding sites that support other browsers but don't include Edge
+*/
+
+void function () {
+    window.CSSUsage.StyleWalker.recipesToRun.push(function browserMentions(element, results) {
+        var str = element.innerText.toLowerCase();
```
Comment (gregwhitworth) [inline src/recipes/browserMentions.js]: remember you don't want to run `toLowerCase()` on something that may be `null`. So ensure to check this first before running a method on `str`.

File: tests/recipes/browserMentions.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <link type="text/css" href="../test-page/style.css" media="all" rel="stylesheet" />
+    <script src="../../Recipe.min.js"></script>
+    <title>Supported Browsers pAge</title>
```
Comment (gregwhitworth) [inline tests/recipes/browserMentions.html]: Fix the casing in `pAge`

**Comment by gregwhitworth** [comment]
Since this work is completed and we only merge in archived recipes we wish to run again, closing this out.

**Pull request closed without merging**